sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -18,6 +18,288 @@
 void enableZeTracing();
 void disableZeTracing();
 
+// TODO: Move these functions to the UR plugin once it's updated to include the
+// changes from oneapi-src/unified-runtime#849. urEnqueueCooperativeKernelLaunch
+// llvm/sycl/plugins/unified_runtime/ur/adapters/level_zero/kernel.cpp
+namespace pi2ur {
+pi_result piextKernelSuggestMaxCooperativeGroupCount(pi_kernel Kernel,
```

Comment:
thanks @0x12CC . this file is just a shim layer point redirecting to unified runtime. Any new PI interface therefore needs to have a UR interface (https://github.com/oneapi-src/unified-runtime), and implementation should go directly in unified_runtime/ur/adapters/level_zero.